### ISSUE TITLE
Unwanted white background

### ISSUE THREAD

**Issue by ArshadSiddiqui** [zenozeng/p5.js-svg#235]
SVG has an extra ```<g>``` tag with white fill which is not part of the drawing.
Run this script code to see the bug:

```
function setup() {
  createCanvas(100, 100, SVG);
  resizeCanvas(110, 100)
}

function draw() {
  clear()
}
```
Or see the working code here: https://replit.com/@arshadsid/p5-svg-white-bg-bug

Bug: SVG background is white.
Expected: SVG background should be transparent.

**Comment by zenozeng** [comment]
@ArshadSiddiqui Hi, this bug should be fixed in v1.5.1. If you have some spare time, you can give it a try.

**Comment by ArshadSiddiqui** [comment]
Great, it works! thank you!!
